src/components/SEO/index.tsx

```diff
@@ -72,18 +76,28 @@ const SEO: React.FC<ISEOProps> = ({
   }
 
   if (image) {
+    const imageUrl = siteMeta.siteUrl + image.src
+
     prebuildMeta.push(
       {
         property: 'og:image',
-        content: image
+        content: imageUrl
       },
       {
         property: 'og:image:secure_url',
-        content: image
+        content: imageUrl
+      },
+      {
+        property: 'og:image:width',
+        content: String(image.presentationWidth)
+      },
+      {
+        property: 'og:image:height',
+        content: String(image.presentationHeight)
       },
       {
         name: 'twitter:image',
-        content: encodeURI(`${siteMeta.siteUrl}${image}`)
```

Comment:
was `encodeURI` actually needed? cc @iAdramelk ?

Reply:
To be hones I'm not sure. The one who originally added it to the blog was you https://github.com/iterative/blog/commit/3b5c975363fd4f86ef66dd102a2a130759df8fcb . I think then that you where fixing some kind of bug that we have without escaping, but if not, I'm not sure that we need it.

Reply:
Good question. I see that I was solving a different problem in the first place - absolute URL vs relative one. Don't remember at all why did I put `encodeURI` to be honest.